pages/common/mongoimport.md

```diff
@@ -0,0 +1,24 @@
+# mongoimport
+
+> Imports content from a JSON, CSV, or TSV file into a MongoDB.
+> More information: <https://docs.mongodb.com/database-tools/mongoimport/>.
+
+- Import a JSON file into a specific collection:
+
+`mongoimport --file={{path/to/file}} --uri={{mongodb_uri}} --collection={{collection_name}}`
+
+- Import a CSV file using the first line as the field list into the localhost instance:
+
+`mongoimport --type={{csv}} --file{{path/to/file}} --db={{database_name}} --collection={{collection_name}}`
+
+- Import a JSON array using each element as a MongoDB document:
+
+`mongoimport --jsonArray --file={{path/to/file}}`
+
+- Import a JSON file using a specific mode and a query to match already existing documents:
+
+`mongoimport --file={{path/to/file}} --mode={{delete|merge|upsert}} --upsertFields="{{field1,field2,...}}"`
```

Comment:
For those of you who don't know what "upsert" means 😉 it's a combination of "insert" and "update".  It updates the document, or inserts a new one if it doesn't already exist.

```suggestion
`mongoimport --file={{path/to/file.json}} --mode={{delete|merge|upsert}} --upsertFields="{{field1,field2,...}}"`
```

Reply:
(https://github.com/tldr-pages/tldr/pull/6008#discussion_r636840880) It just his to be in JSON format, it doesn't actually hahaves to be a `.json` file.

Reply:
Also, is not a valid JSON file, because MongoDB expects 1 document per line (we can call them line-separated objects). For a JSON array you should use `--jsonArray` flag, otherwise is invalid.

Reply:
I saw that comment, but if it usually takes a JSON file, why not indicate that?  I haven't seen a tool yet that will actually break if the extension isn't a specific one.

There's a standard for that @navarroaxel: https://jsonlines.org/

Reply:
Yeah, for MongoDB, not for Node :grimacing:

Reply:
If it's 1 JSON  object per line, then the correct file extension is [`.jsonl`](https://jsonlines.org/).